Sources/UI/Camera/Camera.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  Camera.swift
+//  PovioKit
+//
+//  Created by Ndriqim Nagavci on 13/10/2022.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import AVFoundation
+import PovioKit
+
+public class Camera: NSObject {
+  var device: AVCaptureDevice? {
+    switch cameraPosition {
+    case .back:
+      return isCameraAvailable(position: .back) ? AVCaptureDevice.default(for: .video) : nil
+    case .front:
+      return isCameraAvailable(position: .front) ? AVCaptureDevice.default(deviceType, for: .video, position: .front) : nil
+    }
+  }
+  let session = AVCaptureSession()
+  // Communicate with the session and other session objects on this queue.
+  let sessionQueue = DispatchQueue(label: "com.poviokit.camera")
+  public lazy var previewLayer: AVCaptureVideoPreviewLayer = {
+    let previewLayer = AVCaptureVideoPreviewLayer(session: session)
+    previewLayer.videoGravity = AVLayerVideoGravity.resizeAspectFill
+    
+    return previewLayer
+  }()
+  public let cameraService: CameraService
+  public var deviceType: AVCaptureDevice.DeviceType = .builtInWideAngleCamera
+  var cameraPosition: CameraPosition = .back
```

Comment:
We have a setter `setCameraPosition`. Can make this readonly?

Reply:
`setCameraPosition` is part of `PhotoCamera`, and every time the camera position is changed we need to run `configureComponents` on `PhotoCamera` so I would not make this readonly

Reply:
My bad 👌